### PR TITLE
cc33xx-target-scripts: add debian/ and support files

### PR DESCRIPTION
**Pull request by jsuhaas22**
cc33xx-target-scripts are a bunch of config files and scripts to use and control cc33xx WLAN devices.

Add debian/ and support files for it.

Based on Yocto recipe:
https://git.ti.com/cgit/arago-project/meta-ti/commit/?h=scarthgap&id=9c03a91c81b36f01fc1bfecdd726de705a86c993

### CODE REVIEW

File: cc33xx-target-scripts/suite/trixie/debian/copyright [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+           drv/set_power_mode.sh
+           drv/unload_cc33xx.sh
+           sta/sta_add.sh
+           sta/sta_connect-ex-dhcp.sh
+           sta/sta_connect-ex.sh
+           sta/sta_connect.sh
+           sta/sta_remove.sh
+           sta/sta_start.sh
+           sta/sta_stop.sh
+Copyright: __NO_COPYRIGHT_NOR_LICENSE__
```
Comment (glneo) [inline cc33xx-target-scripts/suite/trixie/debian/copyright]: All these should be covered under the license in the COPYING file, which looks like a standard 3-clause BSD.

Comment (jsuhaas22) [reply cc33xx-target-scripts/suite/trixie/debian/copyright]: Its TI-TFL. So have pushed the (hopefully) corrected copyright.

File: cc33xx-target-scripts/suite/trixie/debian/control [hunk 2]
```diff
@@ -0,0 +1,14 @@
+Source: cc33xx-target-scripts
+Section: utils
+Priority: optional
+Maintainer: Suhaas Joshi <s-joshi@ti.com>
+Build-Depends: debhelper-compat (= 13)
+Standards-Version: 4.5.0
+Homepage: https://www.ti.com/tool/CC33XX-SOFTWARE
+
+Package: cc33xx-target-scripts
+Architecture: arm64
```
Comment (glneo) [inline cc33xx-target-scripts/suite/trixie/debian/control]: Are these just scripts? If so they should not be arch dependent.

Comment (jsuhaas22) [reply cc33xx-target-scripts/suite/trixie/debian/control]: Made them arch-independent.

File: cc33xx-target-scripts/suite/trixie/debian/copyright [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+           drv/set_lsi.sh
+           drv/set_power_mode.sh
+           drv/unload_cc33xx.sh
+           sta/sta_add.sh
+           sta/sta_connect-ex-dhcp.sh
+           sta/sta_connect-ex.sh
+           sta/sta_connect.sh
+           sta/sta_remove.sh
+           sta/sta_start.sh
+           sta/sta_stop.sh
```
Comment (glneo) [inline cc33xx-target-scripts/suite/trixie/debian/copyright]: You could probably use wildcard* here to shorten this list.

File: cc33xx-target-scripts/suite/trixie/debian/copyright [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+  * special, exemplary, or consequential damages (including, but not
+  * limited to, procurement of substitute goods or services; loss of use,
+  * data, or profits; or business interruption) however caused and on any
+  * theory of liability, whether in contract, strict liability, or tort
+  * (including negligence or otherwise) arising in any way out of the use
+  * of this software, even if advised of the possibility of such damage.
+  */
+ .
+
+Files:     conf/hostapd.conf
```
Comment (glneo) [inline cc33xx-target-scripts/suite/trixie/debian/copyright]: Why is this one file a different license? Why does a config file have a license at all.. Could you see if the CC33xx folks can fix this up?

File: cc33xx-target-scripts/suite/trixie/debian/copyright (hunk 3 above)
Comment (jsuhaas22) [reply cc33xx-target-scripts/suite/trixie/debian/copyright]: Done.

File: cc33xx-target-scripts/suite/trixie/debian/copyright (hunk 4 above)
Comment (jsuhaas22) [reply cc33xx-target-scripts/suite/trixie/debian/copyright]: I have asked them. Will update this once they get back.

Comment (jsuhaas22) [reply cc33xx-target-scripts/suite/trixie/debian/copyright]: @glneo This has been fixed upstream, so I have removed hostapd.conf from the copyright.

**Review by glneo — APPROVED** [review]

**Pull request merged**